packages/config/src/options.js

```diff
@@ -281,11 +281,17 @@ export function getNuxtConfig(_options) {
     consola.warn('vendor has been deprecated due to webpack4 optimization')
   }
 
+  if (options.build && options.build.parallel && typeof options.build.extractCSS !== 'undefined') {
+    delete options.build.extractCSS
```

Comment:
This may be simple and no need to other checks
```suggestion
options.build.extractCSS = false
```